docs/readme/examples_source/postprocessing/predict.md

```diff
@@ -4,15 +4,14 @@
 
 [comment]:postprocessor-pred-start
 ```python
-import torch
 
 from oml.datasets import ImageQueryGalleryDataset
 from oml.inference import inference
 from oml.models import ConcatSiamese, ViTExtractor
 from oml.registry.transforms import get_transforms_for_pretrained
 from oml.retrieval.postprocessors.pairwise import PairwiseReranker
 from oml.utils.download_mock_dataset import download_mock_dataset
-from oml.utils.misc_torch import pairwise_dist
+from oml.retrieval import RetrievalResults
```

Comment:
[nit] looks like this import should be a few lines above after sorting. Maybe not only in this file